pages/event/[eventSlug]/preScreeningMiddle/index.vue

```
@@ -0,0 +1,150 @@
+          Para continuar a inscrição, responda o questionário e verifique se você pode realizar a doação.
+        </p>
+  
+        <p class="note">
+          Este questionário serve como uma orientação inicial com perguntas frequentes sobre a doação,
+          mas não substitui a triagem realizada por profissionais de saúde no dia e no local da doação.
+        </p>
+  
+        <NuxtImg src="images/donation-illustration.svg" alt="Ilustração de Doação" class="illustration" />
+      </main>
+      <PreScreeningMiddleFooter 
+      :event-slug="eventSlug" />
```

ESLint

[error] Delete `⏎······`

(prettier/prettier)